Sources/AppStoreConnectCLI/Commands/TestFlight/Builds/Localizations/CreateBuildLocalizationsCommand.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+
+struct CreateBuildLocalizationsCommand: CommonParsableCommand, CreateUpdateBuildLocalizationCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "create",
+        abstract: "Create localized What’s New text for a build."
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @OptionGroup()
+    var build: BuildArguments
+
+    @OptionGroup()
+    var localization: BuildLocalizationInputArguments
+
+    func validate() throws {
+        try validateWhatsNewInput()
+    }
+
+    func run() throws {
+        let service = try makeService()
+
+        let buildLocalization = try service.createBuildLocalization(
+            bundleId: build.bundleId,
+            buildNumber: build.buildNumber,
+            preReleaseVersion: build.preReleaseVersion,
+            locale: localization.locale,
+            whatsNew: whatsNew
+        )
+
+        // Render long text by SwiftyTable is not supported yet
+        [buildLocalization].render(format: common.outputFormat == .table ? .json : common.outputFormat)
```

Comment:
🤔 should we throw an error that table isn't supported instead of rendering in a different format?

Reply:
Potentially as part of the validation step now that I'm thinking on it

Reply:
As `table` is the default format, it will be a bit user-unfriendly if we just throw error. 
What do you think of not using CommonOptions in this command, and extract these out 
```
 @OptionGroup()
var authOptions: AuthOptions

@Flag(default: .json, help: "Display results in specified format.")
var outputFormat: OutputFormat
```

Reply:
What's the actual issue? The text is too long for the table? Maybe we can just truncate the text when rendering as a table?

Reply:
> What's the actual issue? The text is too long for the table? Maybe we can just truncate the text when rendering as a table?

The table doesn't support line break and long text will break the table structure.
<img width="50%" height="50%" src="https://user-images.githubusercontent.com/30752103/91793708-4a8b2200-ec5c-11ea-94ea-f2c30e81a524.png"/>

Reply:
The table is just meant to be a quick reference. To do any real automation/work we use an encodable format. I think we should be able to filter out the line breaks (replace with space) and truncate in this use case. We can probably alter the title of What's New the table to explain it's not the verbatim what's new.

Something like `What's New (truncated)`